src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>Microsoft.Extensions.Configuration (appsettings.json) support for Serilog.</Description>
     <VersionPrefix>3.1.1</VersionPrefix>
+    <LangVersion>latest</LangVersion>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>netstandard2.0;net451;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net451;net452;net461</TargetFrameworks>
```

Comment:
What's the reason for the additional target framework here?

Reply:
I added this target because I saw references to it in the conditions.